docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -126,17 +145,24 @@ Every app requires default endpoints for the App Registry Service. This service
 To be able to use your Mini Framework as an app, add a new Spryker module:
 
 ```bash
-docker/sdk cli composer require spryker/app-kernel spryker/message-broker-aws spryker/propel-encryption-behavior 
+docker/sdk cli composer require 'spryker/app-kernel:dev-feature/without-encryption\ as\ 0.1.0  spryker/message-broker-aws'
 ```
 
 The `spryker/app-kernel` module transforms the Mini-Framework into an app. It provides SyncAPI schema and code for configuration and disconnection, as well as an AsyncAPI schema and code for the AppConfigure and AppDisconnect messages. 
 The `spryker/message-broker-aws` module installs the necessary plugins for sending and receiving messages. 
-After installing the modules, you need to configure them. See the [configuration example](https://github.com/spryker-projects/mini-framework/blob/examples/acp/hello-world/my-app-final/config/Shared/config_default.php#L28).
+
+### Step validation
+
+- Make sure that new packages are added in `composer.json` and composer command is executed without errors.
```

Comment:
```suggestion
Make sure that new packages are added in `composer.json` and composer command is executed without errors.
```shell
docker/sdk cli composer validate
```
```

Reply:
All the step validation are marked as list point